app/components/maps/LoadedReplays.tsx

```diff
@@ -29,7 +30,7 @@ interface LoadedReplaysProps {
 }
 
 const LoadedReplay = ({
-    replay, followed, setFollowed, timeLineGlobal,
+    replay, followed, setFollowed, setHovered, timeLineGlobal,
```

Comment:
I'm not the biggest fan of using the state setter syntax in a props context - when I see `setSomething` I immediately assume it's part of this component's state.
Might just be my bias cause I'm so used to it - but we could change this to `updateFollowed` and `updateHovered` or something similar.
@MKuijpers What's your take?